bcda/auth/ssas_test.go

```diff
@@ -274,7 +274,7 @@ func (s *SSASPluginTestSuite) TestAuthorizeAccessErrIsNilWhenHappyPath() {
 	c, err := client.NewSSASClient()
 	require.NotNil(s.T(), c, sSasClientErrorMsg, err)
 	s.p = SSASPlugin{client: c, repository: s.r}
-	err = s.p.AuthorizeAccess(tokenString)
+	_, _, err = AuthorizeAccess(tokenString)
```

Comment:
Just looked into blank identifiers -- makes sense! Just curious re: why it's preferred instead of the previous version here.

Reply:
Previously this methid only returned an error (or nil). I updated the method to return three objects, but for the test we are only interested in checking if there's an error returned 🙂